packages/app/src/app/overmind/effects/gql/dashboard/queries.ts

```diff
@@ -127,6 +131,33 @@ export const getCollections: Query<
   ${sidebarCollectionDashboard}
 `;
 
+export const getRepos: Query<GetReposQuery, GetReposQueryVariables> = gql`
```

Comment:
Would be good to call this `getPersonalRepos`, makes it more clear that there are two (also a team one).

Reply:
good point! Done :)